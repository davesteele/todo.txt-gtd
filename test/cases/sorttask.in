# ExistingProj
#

Existing task @computer +ExistingProj

# _None
#

New task @computer +ExistingProj
