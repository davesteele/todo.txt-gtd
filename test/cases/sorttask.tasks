
@computer
Existing task @computer +ExistingProj
New task @computer +ExistingProj
