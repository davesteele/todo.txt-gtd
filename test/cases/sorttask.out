# ExistingProj
#

Existing task @computer +ExistingProj

New task @computer +ExistingProj

# _None
#

